package.json

```diff
@@ -2202,6 +2202,47 @@
 						"scope": "window",
 						"order": 12
 					},
+					"gitlens.graph.scrollMarkers.enabled": {
+						"type": "boolean",
+						"default": true,
+						"markdownDescription": "Specifies whether to show markers on the scrollbar in the _Commit Graph_",
+						"scope": "window",
+						"order": 13
+					},
+					"gitlens.graph.scrollMarkers.types": {
+						"type": "array",
+						"default": [
+							"selection",
+							"head"
```

Comment:
Add `"highlights"` here

Reply:
Maybe `localBranches` too

Reply:
Done in commit: [Pluralizes some marker names](https://github.com/gitkraken/vscode-gitlens/pull/2489/commits/a52b4818373288955a200db90b50b72ba210eb97)